lms/djangoapps/bookmarks/api.py

```diff
@@ -0,0 +1,27 @@
+"""
+This File contains bookmark api method(s).
+"""
+
+from bookmarks.serializers import BookmarkSerializer
+from bookmarks.models import Bookmark
+
+
+def get_bookmark(user, usage_key, fields=None, serialized=False):
```

Comment:
As we are building Xblock Service & this method would be used by that service and this API. That's why we decided to put this method in this file.